src/openrct2/rct2/S6Importer.cpp

```diff
@@ -429,7 +429,7 @@ extern "C"
         {
             s6Importer->LoadSavedGame(path);
         }
-        return s6Importer->HasInvalidObjects();
+        return !s6Importer->HasNoInvalidObjects();
```

Comment:
Yeah, good point. I think I swapped back and forth so many times on the sense of these functions that it all got a bit confused.